packages/cli/test/integration.js

```diff
@@ -2692,7 +2692,7 @@ test('deploy a Lambda with 3 seconds of maxDuration', async t => {
   );
 
   // Should not time out
-  url.pathname = '/api/wait-for/2';
+  url.pathname = '/api/wait-for/1';
```

Comment:
same for this, but 0?

Reply:
Probably best to test some duration to make sure it doesn't timeout prematurely